package.json

```diff
@@ -1,98 +1,79 @@
 {
   "name": "emberfire",
-  "description": "The officially supported Ember binding for Firebase",
-  "version": "2.0.9",
-  "author": "Firebase (https://firebase.google.com/)",
-  "homepage": "https://github.com/firebase/emberfire/",
+  "version": "3.0.0-rc.1",
+  "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": "",
   "scripts": {
-    "start": "ember server",
     "build": "ember build",
-    "test": "ember try:testall",
-    "legacy": "gulp legacy"
+    "server": "ember server",
+    "start": "node ./express.js",
+    "all": "npm run ts:precompile && npm run build && npm run start",
+    "package": "npm run ts:precompile && npm run build && npm pack .",
+    "test": "COVERAGE=true ember test",
+    "ts:precompile": "ember ts:precompile",
+    "reference": "typedoc --options typedoc.js addon"
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/firebase/emberfire.git"
+  "dependencies": {
+    "broccoli-file-creator": "^2.1.1",
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^3.0.0",
+    "ember-auto-import": "^1.2.19",
+    "ember-cli-babel": "^6.14.1",
+    "firebase": "^5.8.2"
   },
-  "bugs": {
-    "url": "https://github.com/firebase/emberfire/issues"
+  "devDependencies": {
+    "@types/ember": "^2.8.25",
+    "@types/ember-data": "^2.14.17",
+    "broccoli-asset-rev": "^2.7.0",
+    "codeclimate-test-reporter": "^0.5.0",
+    "ember-ajax": "^3.1.0",
+    "ember-cli": "~3.1.4",
+    "ember-cli-code-coverage": "^0.4.2",
+    "ember-cli-dependency-checker": "^2.1.1",
+    "ember-cli-eslint": "^4.2.3",
+    "ember-cli-fastboot": "^1.1.4-beta.1",
+    "ember-cli-htmlbars": "^2.0.3",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
+    "ember-cli-inject-live-reload": "^1.7.0",
+    "ember-cli-qunit": "^4.3.2",
+    "ember-cli-shims": "^1.2.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-typescript": "^1.3.2",
+    "ember-cli-uglify": "^2.1.0",
+    "ember-data": "~3.1.1",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.0",
+    "ember-fetch": "^5.0.0",
+    "ember-load-initializers": "^1.1.0",
+    "ember-resolver": "^4.5.5",
+    "ember-simple-auth": "^1.6.0",
+    "ember-source": "~3.2.0",
```

Comment:
Thank you! Great to see that our favourite library is moving forward. Great moment. :) I'm looking forward to testing with our popular [Ember.js tutorial](https://yoember.com) app.

If things are going well maybe we could try out to upgrade the addon to Ember 3.4 LTS or maybe Ember 3.7 Latest? What do you think?

Reply:
I'll bump to @latest here and rely on .ember-try for LTS.